### DISCUSSION TITLE
webgl error

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#643] [Q&A] [answered]
now,i set osgb to 3dtiles and load to imodeljs and show success,but
i zoom the view will show error (webgl lost context )

![image](https://user-images.githubusercontent.com/18022720/105625781-63dc7700-5e66-11eb-885e-09facc980367.png)

who can help me ?

**Comment by pmconne** [comment, marked as answer]
It sounds like your 3d tiles contain lots of geometry and/or large textures that are exhausting GPU memory. Use the chrome dev tools' Task Manager to monitor GPU memory usage to find out.

**Comment by paprophet** [comment]
can you gei me a tool for osgb to 3dtiles ?

**Reply by pmconne** [reply]
It depends. Where did your osgb file originate - from ContextCapture?

**Reply by paprophet** [reply]
yes,and i have 3mxb the format too.
now i use the tools from http://www.cesiumlab.com/ 
but the error is now，thank you!

**Reply by pmconne** [reply]
> but the error is now，thank you!

Can I assume you mean the error is **gone** now?

**Reply by paprophet** [reply]
> > but the error is now，thank you!
> 
> Can I assume you mean the error is **gone** now?

the error is exist

**Reply by pmconne** [reply]
Ok...I made a suggestion in my initial reply - did you pursue it? With what little information you've provided I am limited in what other help I can provide.

**Reply by paprophet** [reply]
the error is exist, because i convert a new osgb files to 3dtiles

**Reply by pmconne** [reply]
I made the following suggestion:

> It sounds like your 3d tiles contain lots of geometry and/or large textures that are exhausting GPU memory. Use the chrome dev tools' Task Manager to monitor GPU memory usage to find out.

Did you take this suggestion? If so, what did you find?

**Reply by paprophet** [reply]
ok,in Task Manager to monitor GPU memory , and the gpu memory exceeding the limit capacity.
but i want a tool to handle the osgb files （in display or size  ..）,and to 3mtiles !

**Reply by pmconne** [reply]
We can't increase the amount of GPU memory you have available, so you must decrease the size of the tiles being produced. If you're using http://www.cesiumlab.com to produce the 3d tiles, you should contact them. Otherwise, you should tell us what tools you are using to produce the 3d tiles.
